bin/cli-flags.js

```diff
@@ -106,7 +115,6 @@ module.exports = {
       describe: 'Enable gzip compression',
       group: RESPONSE_GROUP,
     },
-    // findPort is currently not set up
```

Comment:
why did you comment out?

Reply:
The TODO was resolved in #2845.

Reply:
@ylemkimon Can we remove this?

Reply:
@evilebottnawi Do you mean removing `port` from `options`?

Reply:
Oh, my mistake, all fine